### PR TITLE
[lunasvg] Update 2.3.8

### PR DESCRIPTION
**Pull request by sakiodre**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [ ] The "supports" clause reflects platforms that may be fixed by this new version *(not sure what this means)*
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by sakiodre** [comment]
@microsoft-github-policy-service agree

File: ports/lunasvg/fix-install.patch [hunk 1]
```diff
@@ -2,10 +2,10 @@
  1 file changed, 2 insertions(+), 13 deletions(-)
 
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 9916c67..159f3a6 100755
+index ea4b2d2..b6fcbec 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -24,16 +24,5 @@ if(LUNASVG_BUILD_EXAMPLES)
+@@ -23,16 +23,5 @@ if(LUNASVG_BUILD_EXAMPLES)
```
Comment (dg0yt) [inline ports/lunasvg/fix-install.patch:8]: If there is no other change, maybe leave the patch unchanged?

File: ports/lunasvg/portfile.cmake [hunk 2]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
   OUT_SOURCE_PATH SOURCE_PATH
   REPO sammycage/lunasvg
-  REF e612abda858b53160041381a23422cd2b4f42fbd #2.3.1
-  SHA512 44f5d013d918cb5af90114a12857bdd2c204caff761516ef98b12b08d8b6215e91f6d963c281500c386f287b9d0ecd5b3d986d4c8c33423c0c34d539d744e09d
+  REF 87137e791c912432b93982722a8e965628950ca7 #2.3.8
```
Comment (dg0yt) [inline ports/lunasvg/portfile.cmake:4]:
```suggestion
  REF "v${VERSION}
```
... which will have a different SHA512.

File: ports/lunasvg/fix-install.patch (hunk 1 above)
Comment (sakiodre) [reply ports/lunasvg/fix-install.patch:8]: My mistake, I thought since the CMakelist.txt has changed, the patch also needed to be updated

**Review by BillyONeal — DISMISSED** [review]

**Review by FrankXie05 — DISMISSED** [review]

File: ports/lunasvg/portfile.cmake [hunk 3]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
   OUT_SOURCE_PATH SOURCE_PATH
   REPO sammycage/lunasvg
-  REF e612abda858b53160041381a23422cd2b4f42fbd #2.3.1
-  SHA512 44f5d013d918cb5af90114a12857bdd2c204caff761516ef98b12b08d8b6215e91f6d963c281500c386f287b9d0ecd5b3d986d4c8c33423c0c34d539d744e09d
+  REF v${VERSION}
```
Comment (FrankXie05) [inline ports/lunasvg/portfile.cmake:4]:
```suggestion
  REF "v${VERSION}"

```
Not necessary, but more secure.

File: ports/lunasvg/portfile.cmake [hunk 4]
```diff
@@ -1,8 +1,9 @@
 vcpkg_from_github(
   OUT_SOURCE_PATH SOURCE_PATH
   REPO sammycage/lunasvg
-  REF e612abda858b53160041381a23422cd2b4f42fbd #2.3.1
-  SHA512 44f5d013d918cb5af90114a12857bdd2c204caff761516ef98b12b08d8b6215e91f6d963c281500c386f287b9d0ecd5b3d986d4c8c33423c0c34d539d744e09d
+  REF "v${VERSION}"
+
+  SHA512 368f76ae3c04fbcb08406d9e7793af37b5e28ab90ffe0978fae8783620a45af3270fcd4e895c553e3e651f0ba07e37355acffd9f0aab1bf9ef822f465de21073
```
Comment (FrankXie05) [inline ports/lunasvg/portfile.cmake:6]:
```suggestion
  REF "v${VERSION}"
  SHA512 368f76ae3c04fbcb08406d9e7793af37b5e28ab90ffe0978fae8783620a45af3270fcd4e895c553e3e651f0ba07e37355acffd9f0aab1bf9ef822f465de21073

```

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

**Comment by FrankXie05** [comment]
Every change requires running the command `./vcpkg x-add-version lunasvg --overwrite-version`. :)

File: ports/lunasvg/portfile.cmake (hunk 4 above)
Comment (FrankXie05) [reply ports/lunasvg/portfile.cmake:6]: Remove the space.

**Review by FrankXie05 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**